### DISCUSSION TITLE
Readme update

### DISCUSSION THREAD

**Discussion by iplayfast** [open-webui/open-webui#6569] [General]
# Feature Request
when I go to open-webui on my computer, it has a little popup saying "A new version (v0.3.35) is now available."
## Important Notes
1. How do I know what version I'm currently running. 
2. How do I upgrade it. 

For example, I installed it with:
docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main
from the readme.md (ollama already on my computer)

I've tried docker stop, docker rm, and then the above command, but the new version popup still shows up.
*** edit ***, apparently the popup was from the older version(I don't know why it showed), once I logged in however, it knew it was v 3.35.

**Comment by f16-irk** [comment]
HI!
You can go to "Settings - About", there is answer for Q.1.
For upgrade you'll need to use latest image of open-webui.

**Comment by unicorn667** [comment]
# Pull the latest image
docker pull ghcr.io/open-webui/open-webui:main

# Stop and remove the current container
docker stop open-webui

# remove old container 
docker rm open-webui  

# Run the new container with the same parameters
docker run -d -p 3000:8080 \
  --add-host=host.docker.internal:host-gateway \
  -v open-webui:/app/backend/data \
  --name open-webui \
  --restart always \
  ghcr.io/open-webui/open-webui:main

**Reply by iplayfast** [reply]
If I've modifed a model, will the changes be lost if I upgrade?
